exporter/datasetexporter/README.md

```diff
@@ -24,6 +24,14 @@ See the [Getting Started](https://app.scalyr.com/help/getting-started) guide.
 
 If you do not want to specify `api_key` in the file, you can use the [builtin functionality](https://opentelemetry.io/docs/collector/configuration/#configuration-environment-variables) and use `api_key: ${env:DATASET_API_KEY}`.
 
+### Server Host Settings
+
+Specifying server host is important for the correct functionality. DataSet expects that 
```

Comment:
I think it would be a good idea to document each specific scenario with config option and example event payload - just to make it more clear to the end user how the DataSet event will look like.